### ISSUE TITLE
Project lacks a code of conduct

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization/governance#9]
I find myself wondering if the Open Organization project and community would benefit from a publicly expressed code of conduct. In my mind, it'd be part of our community's overall set of governance documents. Opening this issue to see what others think.

**Comment by funnelfiasco** [comment]
We absolutely should have one. I propose using the [Contributor Covenant](https://www.contributor-covenant.org/version/2/0/code_of_conduct/), but I'm open to other ideas.

One issue may be getting our sponsor to sign off on it. In another community, we've been blocked by getting time from Legal for...a while.

**Comment by semioticrobotic** [comment]
Interesting, @funnelfiasco. I had not anticipated that this would be an issue. Let's connect about it so I can hear more about your experience and what you've learned.

**Comment by LauraHilliger** [comment]
Heh? @funnelfiasco that's crazy pants. I also use the Contributor Covenant quite often. It's a great start and I find it doesnt need loads of remixing at all.

**Comment by semioticrobotic** [comment]
Red Hat's [open source participation guidelines](https://www.redhat.com/en/resources/open-source-participation-guidelines-overview) indicate that the [Contributor Covenant v1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct/) is pre-approved for use in Red Hat-supported upstream projects. I have not yet read this code of conduct, but plan to soon. Does anyone else have experience with it?

**Comment by LauraHilliger** [comment]
yep @semioticrobotic - that's the one Ben and I have used in the past. We need an email addy for people to submit code violations to, but otherwise, I think it's specific enough to be effective and general enough to fit most projects (including ours ;)

**Comment by jimmysjolund** [comment]
I think we should include a CoC even though we (might) be doing fine now. Reading the Red Hat's open source participation guidelines:

> Red Hat-led projects may choose to adopt a code of conduct that has been approved by Red Hat Legal, such as the [Contributor Covenant version 1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct/), but are not required or expected to do so.

Not required or expected to do so ... It's not updated since October 2020 and [Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) is now at version 2.1. If that would be the one we would choose to follow.

**Comment by LauraHilliger** [comment]
i think we should just exec decision this. I added it to the footer of the new-website (which is something we should try to find a working session for :) https://github.com/open-organization/new-website/commit/bc3e56a072b6817fb31b1a9fe27c85c65d9bb71a

**Comment by semioticrobotic** [comment]
I remain cautious about executive decisioning (verb?) something as significant and community-effecting as a code of conduct, but am happy to follow the lead of others in this thread if that seems to be the direction we wish to go. Personally, I certainly support the adoption of a CoC and like the Contributor Covenant.

We do need that email address though, or it's a rather empty and toothless policy. @jimmysjolund, I started this work before I changed organizations, so there *should* be an address like `codeofconduct@theopenorganization.org` already in place somewhere. We will need to:

1. Verify that this is indeed the address (the Community Infrastructure team will know; just send Misc a  note).
2. Edit the forwarding address of that email alias. (So we need to decide: who will receive CoC complaints and be in charge of reviewing and addressing them? It's a potentially non-trivial amount of emotional labor so I don't want us to make the decision lightly.)
3. Add the address to the CoC template.

**Comment by LauraHilliger** [comment]
haha yeah, I just meant "it's been 2 years, we need a CoC, this shouldn't be still hanging!"

**Comment by jimmysjolund** [comment]
The email address is in place and we can forward it to selected community members.

**Comment by semioticrobotic** [comment]
Excellent. Thanks, @jimmysjolund for following up on this for us. Seems like @LauraHilliger has this on the agenda for our June community call, and I'd propose we make it a top priority.

**Comment by semioticrobotic** [comment]
At today's meeting of the Open Organization Ambassadors, we agreed to adopt the Community Covenant as our community's code of conduct. Noting here for @LauraHilliger! :partying_face: 

I also noted my comment above :point_up:. [Contributor Covenant v1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct/) is already pre-approved for Red Hat-sponsored projects as per Red Hat's open source participation guidelines, so I recommend beginning with that. Ultimately, however, the choice of version is the community's.

Glad to see motion on this!
